docs/faq/can-i-host-webiny-on-my-laptop.md

```diff
@@ -0,0 +1,13 @@
+---
+id: can-i-host-webiny-on-my-laptop
+title: Can I host Webiny on my laptop?
+sidebar_label: Can I host Webiny on my laptop?
+---
+
+The short answer is that you are unable to do so.
+
+We have various serverless resources that are deployed to AWS such as Lambda Functions, S3 Buckets and Cognito.
```

Comment:
I think @Pavel910 can comment here on a quick sentence why they are difficult to be created locally.

Reply:
@Fsalker you can actually use parts of this paragraph https://docs.webiny.com/docs/get-started/development-principles#cloud-native-development to explain why it's not possible to run API locally.

Reply:
nice idea